README.md

```diff
@@ -16,6 +18,13 @@ To build Buck, run the following:
     ant
     ./bin/buck --help
 
+<hr/>
+
 License
 -------
 Apache License 2.0
+
+<hr/>
+
+<img src="https://lucene.apache.org/images/mantle-asf.png" width="150"></img>
+<img src="https://www.facebook.com/images/fb_icon_325x325.png" width="75"></img>
```

Comment:
These images don't add anything and aren't necessary.

We are not an apache project so adding an ASF logo doesn't even make sense.